.circleci/config.yml

```diff
@@ -1,48 +1,60 @@
-version: 2
+version: 2.1
+
+executors:
+  node:
+    parameters:
+      browsers:
+        type: boolean
+        default: false
+    docker:
+      - image: circleci/node:latest<<# parameters.browsers >>-browsers<</ parameters.browsers >>
+    working_directory: ~/project
+    environment:
+      NODE_ENV: test
+      NODE_OPTIONS: --max_old_space_size=4096
 
-defaults: &defaults
-  working_directory: ~/project
-  docker:
-    - image: circleci/node:latest
-  environment:
-    NODE_ENV: test
-    NODE_OPTIONS: --max_old_space_size=4096
+commands:
+  attach-project:
+    steps:
+      - checkout
+      - attach_workspace:
+          at: ~/project
+  store-coverage:
```

Comment:
I realize this is probably the wrong name, but I wanted something different then eg `store-test-results`. Let me know if you have a suggestion

Reply:
store-test-results ?